improve README in workflows (#129)
galaxyproject/iwc: workflows/README.md

## Adding workflows

Here are some guidelines to help new contributors to add their workflows.

Everything starts from a workflow that you need to download locally.

Currently the CI is using CVMFS for tools and for references, this is why all tests described below are against usegalaxy.org.

### Optional: Updating tools

It is recommended to check and update tools with Planemo before import :

```bash
planemo autoupdate <workflow.ga>
```

> Then, it is recommanded to upload this updated workflow inside usegalaxy.org server as some updates may break connections (and some tools may not have the last update).

Reviewer: ```suggestion
Then, it is recommended to upload this updated workflow to the usegalaxy.org server as some updates may break connections (and some tools may not yet be installed on usegalaxy.org).
```
